prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -149,6 +151,7 @@ def build_transmission():
     """Main user-facing entry point."""
     # Load input data
     hifld_substations = get_hifld_electric_substations(const.blob_paths["substations"])
+    hifld_substations.set_index("ID", inplace=True)
```

Comment:
I presume this is prepared for future use. Just to confirm (I believe that's the case), `get_hifld_electric_substations` will give us the final list of substations. We won't drop or add any substations after this point, right? But the substation ID still won't be continuous, given we filtered out those out-of-service.

Reply:
The reason we reset the index here is so that we can call `idxmin` within `find_closest_substation_and_distance` and get something meaningful back.

Reply:
Sorry, I mean, this substation ID will be used in the final `sub.csv` as well?

Reply:
Ah! Good question, I don't know yet.